src/main/java/atreides/ui/Response.java

```diff
@@ -0,0 +1,18 @@
+package atreides.ui;
+
+public class Response {
+    String msg;
+    String LINE = "____________________________________________________________";
```

Comment:
Should this variable be a **static final** variable if it is a constant in screaming snake case (all capitals), or renamed if it is not a constant?